UI/src/view/LinearAssetLayer.js

```diff
@@ -171,6 +172,15 @@ window.LinearAssetLayer = function(params) {
   map.addLayer(indicatorLayer);
   indicatorLayer.setVisible(false);
 
+  var trafficSignReadOnlyLayer = new TrafficSignReadOnlyLayer({
```

Comment:
We could have the layer as a parameter instead of initialize the readonly layer inside the linear asset layer. If we do it like that we can easily use another readonly layer in the future.